addons/l10n_au/views/res_partner_bank_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_partner_bank_form" model="ir.ui.view">
+        <field name="name">aba.res.partner.bank.form</field>
+        <field name="model">res.partner.bank</field>
+        <field name='inherit_id' ref='base.view_partner_bank_form'/>
+        <field name="arch" type="xml">
+            <field name="acc_holder_name" position='after'>
+                <field name='aba_bsb'/>
```

Comment:
Shouldn't this field be visible only when `acc_type == 'aba'`?

Reply:
@william-andre  No.  ABA is valid for all bank accounts - it is mandatory if using ABA - but can be used in other contexts.  I was loathe to rename the field, so I kept it as aba_bsb, but in reality, it could have been just bsb

No, I don't believe I have access to the upgrade repo.